app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  after_save :update_post_counter
```

Comment:
Great job @kambereBr 🤝

- [ ] [Optional] Since you have used the after_save to call the `update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method `private`. Kindly apply this to the `comment` and the `like` models.

Reply:
Dear @1GORDON, thanks for your review 👏 

Suggestions applied ✅